src/style.css

```diff
@@ -1,3 +1,72 @@
+/* #region General */
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --primary-color: #00ACDC;
+  --black-color: #000;
 }
+
+/* #endregion */
+
+/* #region Header */
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  line-height: 0;
+}
```

Comment:
Redundant
```suggestion
```

Reply:
Without this property, the "a" tag has an additional 4px of height and will not allow to perform - Content is vertically centered (for any header height). 
At least visually.